ansible_rulebook/engine.py

```diff
@@ -52,6 +52,14 @@
 logger = logging.getLogger(__name__)
 
 
+all_source_queues = []
+
+
+async def broadcast_shutdown(obj):
+    for queue in all_source_queues:
+        queue.put_nowait(obj)
```

Comment:
either remove `async` from the method def or use `queue.put()`.  better to use `queue.put()`